src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/DJApplicantReceivedNotificationHandler.java

```diff
@@ -196,4 +196,10 @@ public String getLegalOrganizationName(String id, CaseData caseData) {
         return caseData.getApplicantSolicitor1ClaimStatementOfTruth().getName();
     }
 
+    private String getLipEmailTemplate(CaseData caseData) {
```

Comment:
We do not need reference to entire caseData object here . Pass Boolean (caseData.isBilingual()) instead please

Reply:
fixed